nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -403,6 +434,9 @@ public void process(final InputStream in) throws IOException {
                                 attributes.put(FRAGMENT_ID, fragmentId);
                                 attributes.put(FRAGMENT_INDEX, String.valueOf(++fragmentCount));
 
+                                final String encryptionMethod = zipEntry.getEncryptionMethod().toString();
+                                attributes.put(FILE_ENCRYPTION_METHOD_ATTRIBUTE, encryptionMethod);
```

Comment:
Can this value be `null` if there is no/unknown encryption method used?

Reply:
The encryptionMethod property defaults to NONE, and all references in the source repository appear to set a value.  Here's the initial declaration: https://github.com/srikanth-lingala/zip4j/blob/master/src/main/java/net/lingala/zip4j/model/AbstractFileHeader.java#L23